gradoop-flink/src/test/java/org/gradoop/flink/model/impl/tpgm/TemporalGraphCollectionTest.java

```diff
@@ -79,16 +88,56 @@ public void testIsEmpty() throws Exception {
   /**
    * Test the {@link TemporalGraphCollection#writeTo(DataSink)} method.
    */
+  @Test
+  public void testWriteTo() throws Exception {
+    String tempFolderPath = testFolder.newFolder().getPath();
+
+    testTemporalCollection.writeTo(new CSVDataSink(tempFolderPath, getConfig()));
+    getExecutionEnvironment().execute();
+
+    DataSource dataSource = new CSVDataSource(tempFolderPath, getConfig());
+
+    collectAndAssertTrue(dataSource
+      .getTemporalGraphCollection()
+      .toGraphCollection()
+      .equalsByGraphElementData(testTemporalCollection.toGraphCollection()));
+  }
+
+  /**
+   * Test the {@link TemporalGraphCollection#writeTo(DataSink, boolean)} method.
```

Comment:
Mention that this tests overwriting.
Also missing `throws` tag for `Exception`.